xrefcheck.nix

```diff
@@ -11,7 +11,20 @@ let
     src = pkgs.haskell-nix.haskellLib.cleanGit { src = ./.; };
     modules = [{
       packages.xrefcheck = {
-        ghcOptions = [ "-Werror" ];
+        ghcOptions =
+          [
+            "-Werror"
+            "-Weverything"
+            "-Wno-missing-safe-haskell-mode"
+            "-Wno-unsafe"
+            "-Wno-missing-import-lists"
+            "-Wno-missing-local-signatures"
+            "-Wno-missing-export-lists"
+            "-Wno-all-missed-specialisations"
+            "-Wno-prepositive-qualified-module"
+            "-Wno-monomorphism-restriction"
```

Comment:
Please move the flags to `package.yaml` - we want them to be turned on at all times, not just in the CI.

The only flag we disable in dev environments is `-Werror`, for convenience.

Reply:
Oh and delete `-Wall`